packages/dev/core/src/Materials/Node/Blocks/PBR/subSurfaceBlock.ts

```diff
@@ -159,7 +171,7 @@ export class SubSurfaceBlock extends NodeMaterialBlock {
             vec2 vThicknessParam = vec2(0., ${thickness});
             vec4 vTintColor = vec4(${tintColor}, ${refractionTintAtDistance});
             vec3 vSubSurfaceIntensity = vec3(${refractionIntensity}, ${translucencyIntensity}, 0.);
-
+            float dispersion = ${dispersion};

```

Comment:
You should issue this line only if dispersion is not `undefined`.

Reply:
Done